docker/README.md

```diff
@@ -0,0 +1,44 @@
+This document is used for generating OWT build and all-in-one OWT running environment image.
+
+## Prequisites
+
+  * Docker is setup in your local environment and related proxy is set so that Docker can access to external network (including Google because OWT will download webrtc related code and patch from Google)
+
+## Build OWT images
+
+1. Run following command to generate Docker image containing OWT source code and building dependencies, and a OWT dist folder will be packed with all OWT running binaries, you can also run OWT service with this image in case you want to modify OWT source code and debug. 
+
+    ```shell
+    docker build --target owt-build -t owt:build \
+            --build-arg http_proxy=${HTTP_PROXY} \
+            --build-arg https_proxy=${HTTPS_PROXY} \
```

Comment:
```suggestion
            --build-arg https_proxy=${HTTPS_PROXY}
```

Reply:
There is a . next line to specify the build context, so "\" is needed here and in build_docker_image.sh